pyflakes/checker.py

```diff
@@ -911,21 +956,22 @@ def ignore(self, node):
         pass
 
     # "stmt" type nodes
-    DELETE = PRINT = FOR = ASYNCFOR = WHILE = IF = WITH = WITHITEM = \
-        ASYNCWITH = ASYNCWITHITEM = RAISE = TRYFINALLY = EXEC = \
-        EXPR = ASSIGN = handleChildren
+    DELETE = PRINT = EXEC = EXPR = RAISE = handleChildrenFlattened
+    ASSIGN = TRYFINALLY = handleChildren
+    FOR = ASYNCFOR = WHILE = IF = WITH = ASYNCWITH = handleChildren
+    WITHITEM = ASYNCWITHITEM = handleChildrenFlattened
```

Comment:
Why not put all of the nodes using `handleChildrenFlattened` together?

Reply:
I have grouped them by type of node in a readible manner, as best I could with this subset of all nodes

I felt the "with-item" nodes should go after the "with" nodes and other branching nodes.